pymongo/collection.py

```diff
@@ -1246,6 +1246,13 @@ def find(self, *args, **kwargs):
 
         .. versionchanged:: 4.0
            Removed the ``modifiers`` option.
+           Empty projections (eg {} or []) for
+           :meth:`~pymongo.collection.Collection.find`, and
+           :meth:`~pymongo.collection.Collection.find_one`
```

Comment:
I think we can remove the "for find and find_one" part here because this is the docs for find.

Reply:
Makes sense. Fixed.